utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -325,8 +325,8 @@ int main(int argc, char **argv)
 
         for (int i = 0; i < ioBufferSize; i++)
         {
-            *inputf16Temp++ = (half)(((float)*inputTemp++) * conversionFactor);
-            *inputf16SecondTemp++ = (half)(((float)*inputSecondTemp++) * conversionFactor);
+            *inputf16Temp++ = static_cast<half>((static_cast<float>(*inputTemp++)) * conversionFactor);
```

Comment:
Lets use Rpp's types consistently for all these casts. Except I believe half works as Rpp16f on host, but may not on hip.